test/db/cmd/cmd_px

```diff
@@ -1,6 +1,7 @@
 NAME=pxr4
 FILE=malloc://1024
 CMDS=<<EOF
+e asm.arch=x86
```

Comment:
also add asm.bits=64, other this will fail on arm32 or ppc32 for example

Reply:
True. I'll test on raspbian 32 someday to find all this kind of issues, as there's no CI for 32 bit platform.